### PR TITLE
Add status related server unit tests

### PR DESCRIPTION
**Pull request by saschagrunert**
- Removed unused files in server/fixtures
- Migrated the inspect_tests.go to ginkgo and increased coverage
- Added listen_unix, metrics, secrets, version and runtime_status tests

### CODE REVIEW

File: server/inspect_test.go [hunk 1]
```diff
@@ -1,220 +1,129 @@
...
-	"github.com/cri-o/cri-o/oci"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/cri-o/cri-o/pkg/storage"
+	"github.com/go-zoo/bone"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
-func TestGetInfo(t *testing.T) {
```
Comment (rhatdan) [inline server/inspect_test.go]: What happened to these tests?

**Comment by saschagrunert** [comment]
Rebased on top of the latest master branch. :)

**Comment by haircommander** [comment]
LGTM but it looks like you need another rebase (and there's one outstanding question)

File: server/inspect_test.go (hunk 1 above)
Comment (saschagrunert) [reply server/inspect_test.go]: Yeah I see we should keep them, I will leave them side-by-side to the new tests.

**Comment by saschagrunert** [comment]
Rebased on top of the latest master branch and preserved the overwritten tests.

**Comment by haircommander** [comment]
@umohnani8 @rhatdan @mrunalp PTAL

**Pull request merged**